src/main/app/src/components/common/Murupolku/MurupolkuFragment.js

```diff
@@ -0,0 +1,100 @@
+import React from 'react';
+import { Link as RouterLink } from 'react-router-dom';
+import { Button, makeStyles } from '@material-ui/core';
+import HomeOutlinedIcon from '@material-ui/icons/HomeOutlined';
+import ArrowForwardIosIcon from '@material-ui/icons/ArrowForwardIos';
+
+import LocalizedLink from '#/src/components/common/LocalizedLink';
+import { colors } from '#/src/colors';
+
+const BREADCRUMB_ICON_SPACING = '14px';
+
+const useStyles = makeStyles((theme) => ({
+  home: {
+    display: 'inline',
+    marginRight: BREADCRUMB_ICON_SPACING,
+    verticalAlign: 'text-bottom',
+    fontSize: '22px',
+  },
+  arrow: {
+    display: 'inline',
+    marginRight: BREADCRUMB_ICON_SPACING,
+    color: colors.lightGrey,
+    fontSize: '12px',
+  },
+  link: ({ isLast, isHome, link }) => ({
+    ...theme.typography.body1,
+    marginRight: BREADCRUMB_ICON_SPACING,
+    display: 'inline',
+    cursor: 'default',
+    '&:hover': {
+      textDecoration: 'none',
+    },
+    ...(link && {
+      cursor: 'pointer',
+      color: colors.green,
+    }),
+    ...(isLast &&
+      !isHome && {
+        color: theme.palette.text.primary,
+        pointerEvents: 'none',
+        fontWeight: 600,
+      }),
```

Comment:
Hmm, eiks spreadi heitä poikkeuksen jos se löytää ei-spreadattavaa? e.g. ...(false)

Reply:
Olis se varmasti tuohon jo kosahtanu :-D. Kyllä spread näyttäis objekteille noin toimivan, taulukoille ilmeisesti ei. Ehkä selkeämpää tehdä kaikki ternarylla.

Reply:
Joo näemmä menee vaan taulukolla ajossa poikkeukseen, Pitäs kattoo miten toi transpilettyy käytännössä niin tietäis mikä ero nois on

Reply:
Laitoin nyt ternarylla kuitenkin.